quantum/mousekey.c

```diff
@@ -288,16 +288,16 @@ static int8_t calc_inertia(int8_t direction, int8_t velocity) {
     // simulate acceleration and deceleration
 
     // deceleration
-    if ((direction > -1) && (velocity < 0))
-        velocity = (velocity + 1) * (256 - MOUSEKEY_FRICTION) / 256;
-    else if ((direction < 1) && (velocity > 0))
+    if (velocity != 0)
         velocity = velocity * (256 - MOUSEKEY_FRICTION) / 256;
 
     // acceleration
     if ((direction > 0) && (velocity < mk_time_to_max))
         velocity++;
     else if ((direction < 0) && (velocity > -mk_time_to_max))
         velocity--;
+    else if (velocity <= MOUSEKEY_STOP_SPEED && velocity >= -MOUSEKEY_STOP_SPEED)
+        velocity = 0;
```

Comment:
I wasn't quite sure what to do here regarding the styleguide. The guide is unambiguous in saying that braces must always be used... but since I am extending an existing unbraced if/else chain it felt wrong to differently format only my lines in the function.

I suppose I could have reformatted the entire function, but I worried that doing so would be considered out-of-scope for the purposes of this PR.